src/cpu/aarch64/acl_deconvolution.hpp

```diff
@@ -88,7 +88,8 @@ struct acl_deconvolution_fwd_t : public primitive_t {
             , acl_pd_conf()
             , post_ops() {}
 
-        DECLARE_COMMON_PD_T("acl", acl_deconvolution_fwd_t);
+        DECLARE_COMMON_PD_T(
+                "acl:deconv", acl_deconvolution_fwd_t, USE_GLOBAL_SCRATCHPAD);
```

Comment:
(nit) other acl impls do not use the primitive kind in name. Any reason to add it here?

Reply:
Taking a quick look, I see both. Sorry that we are not being consistent here. For example, 
indirect_gemm:acl -> for indirect gemm convolution
gemm:acl -> for gemm convolution
gemm:acl -> for matmul
acl -> for reorder

I will make this `deconv:acl` then.

Reply:
yes it seems indirect_gemm and gemm are the two outliers, all others are just "acl".
tagging @jondea for some guidance here.

Reply:
I think the `gemm:acl` impl of matmul is the outlier. The `indirect_gemm` and `gemm` implementations of convolution are distinct implementations of the same primitive, that we would want to distinguish between in the logs.

Or in other words, I think this should stay as `acl` because there is only one `acl` implementation of the deconvolution primitive.

Reply:
Done.